src/test/html/html-script-scanner_test.ts

```diff
@@ -63,11 +64,14 @@ suite('HtmlScriptScanner', () => {
     let analysis: Analysis;
 
     before(async () => {
-      analysis = await analyzer.analyze(['js-modules.html']);
+      analysis = await analyzer.analyze(
+          ['js-modules.html', 'base-href/imports-js-module-with-base.html']);
     });
 
     test('finds external module scripts', () => {
-      const htmlScripts = [...analysis.getFeatures({kind: 'html-script'})];
+      const htmlScripts =
+          [...(analysis.getDocument('js-modules.html') as Document)
```

Comment:
Why get the document here?

Reply:
This comment isn't blocking, btw, but the getDocument call seems unnecessary to me.

Reply:
Because I want to restrict the features to those obtained from the js-modules document instead of across the analysis (which I expanded to include the additional document).